index.html

```diff
@@ -4,7 +4,71 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="style.css" rel="stylesheet" />
```

Comment:
I would highly recommend deleting this line, because you don't have a `style.css` for now and the page would try to load it and would throw an error in the console. You don't want that. ;-) You can add the line later, when you have a `style.css` file.